utils/components.yml

```diff
@@ -1,32 +1,32 @@
 al-sgov-server:
   name: SGoV Service
-  url: https://kbss.felk.cvut.cz/sgov-server
+  url: https://xn--slovnk-test-scb.mvcr.gov.cz/modelujeme/sluzby/sgov-server
   meta: {}
 al-db-server:
   name: A graph database
   url: https://example.com/sparql
```

Comment:
Not sure, why you replaced termit-server URL, but not this one. Would propose to have everything pointed to slovník-test

Reply:
Fixed. It does not really matter as MC does not access neither of those services. The configuration is applied only for local development and on Netlify, or to be precise for generating local env vars. On the main deployment instances it is always replaced by the docker compose configuration.